04.android/06.solving-problems/15.multiple-profiles-issue/docs.en.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Problem with reinstalling AdGuard on Android devices when using the app on more than one user account'
+taxonomy:
+   category:
+       - docs
+visible: true
+---
+ 
+In Android 9 and later, if AdGuard is installed for more than one user account on your device, you may encounter problems with removing the app. When you delete AdGuard from one user account, the app will still show up on the list of applications, but you can’t uninstall or reinstall it. It happens because AdGuard is installed for another user account on the device.
```

Comment:
```suggestion
In Android 9 and later, if AdGuard is installed for more than one user account on your device, you may encounter problems when uninstalling the app. When you uninstall AdGuard from one user account, the app will still appear on the app list, but you won't be able to uninstall or reinstall it. It happens because AdGuard is installed for a different user account on the device.
```